src/lib/utilities/find-value-if-it-includes-word.ts

```diff
@@ -0,0 +1,7 @@
+export const findValueIfItIncludesWord = (object, word) => {
```

Comment:
We already have a more TypeScript-friend (if also more verbose) version of this in `src/lib/utilities/get-component-for-event-type.ts`.